Components/avatar/avatar.html

```diff
@@ -129,28 +130,59 @@ <h4 class="list-topic"> Components</h4>
-                        alt="" class="avatar-image">
-                </div>
 
-                <div class="avatar avatar-md">
-                    <img src="https://images.unsplash.com/photo-1606663889134-b1dedb5ed8b7?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
-                        alt="" class="avatar-image">
+            <!-- Alert Content Start -->
+            <main class="component-content-container">
+
+                <div class=" component">
+                    <h1 class="component-title">Avatar</h1>
+
+                    <p class="component-details">An avatar component can be used to display circular user profile pictures. Avatar can be used to portray people or objects. It supports images, icons, or letters.</p>
+
+                    <h2 class="component-subtitle">Example of Avatars</h2>
+
+                    <p class="component-details">Here are some avatar components for you with examples. If you want to use them in order to develop your apps with a faster pace just add these classes to your code. For different sizes you can use these classes:-
+                        <mark class="text-highlight">avatar-xl</mark>
+                        <mark class="text-highlight">avatar-lg</mark>
+                        <mark class="text-highlight">avatar-md</mark>
+                        <mark class="text-highlight">avatar-sm</mark>
+
+                    </p>
+
+                    <div class="avatar-container">
+                        <div class="avatar">
+                            <img src="https://images.unsplash.com/photo-1593085512500-5d55148d6f0d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=880&q=80"
+                                alt="" class="round-avatar-image avatar-xl">
+                        </div>
+                    
+                        <div class="avatar">
+                            <img src="https://images.unsplash.com/photo-1643224981923-b63f2417d220?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=424&q=80"
+                                alt="" class="round-avatar-image avatar-lg">
+                        </div>
+                    
+                        <div class="avatar">
+                            <img src="https://images.unsplash.com/photo-1606663889134-b1dedb5ed8b7?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+                                alt="" class="round-avatar-image avatar-md">
+                        </div>
+                    
+                        <div class="avatar">
+                            <img src="https://images.unsplash.com/photo-1635586409095-b5d87cebe12b?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
```

Comment:
Always give **alt** (alternate) in (avatar, badge).

Reply:
Added that